org.hl7.fhir.publisher.core/src/main/java/org/hl7/fhir/igtools/web/PublicationProcess.java

```diff
@@ -615,7 +615,7 @@ private void doPublish(File fSource, File fOutput, JsonObject qa, String destina
               if (!v.cibuild() && !v.current()) {
                 String pv = v.path();
                 String vCode = pv.substring(pv.lastIndexOf("/")+1);
-                String dv = Utilities.path(destination, vCode);
+                String dv = Utilities.path(fRoot, relPath);

```

Comment:
I feel like dv might actually be 'path' or some other combo of these path manipulations already, but I can't quite hit it in debug traces.